lib/ipUtils/cidr.go

```diff
@@ -48,10 +48,10 @@ func (c *CIDR) Size() int {
 }
 
 // list returns a slice of sorted CIDR structures.
```

Comment:
All these comments need to be fixed to capitalize the function. All!

Reply:
Many of them are already capital

Reply:
The ones which were remaining I have changed them